### PR TITLE
Standard & Fibonacci pivot points for trading levels implementation added.

### PR DESCRIPTION
**Pull request by Ahsan9981**
I have added standard & Fibonacci pivot point levels for trading. Input involve candle (OHLC values) and type of pivot points wanted and output is calculated pivot points as per input.

### CODE REVIEW

**Comment by Vasile-Peste** [comment]
Wow, I will check as soon as possible, thank you!

**Comment by Ahsan9981** [comment]
Sure, please check and let me know what i did wrong. I think i am not a
very good programmer and still need alot of improvements.

**Comment by Ahsan9981** [comment]
I was having hard time wrapping my head around the base indicator class as input and outputs are different.
Previously implemented indicators used array of numbers as input and output was a single value. where as I used candle object and pivot point objects. There are different type of pivot points so I have implemented an abstract class  and extended depending it depending upon situation.

**Comment by Vasile-Peste** [comment]
> I was having hard time wrapping my head around the base indicator class as input and outputs are different.
> Previously implemented indicators used array of numbers as input and output was a single value. where as I used candle object and pivot point objects. There are different type of pivot points so I have implemented an abstract class and extended depending upon situation.

The idea is that a base indicator can have different types of input/output, so when you implement your indicator, you choose in the specific case which data type is more suitable as input for it and document it accordingly. However you have done a great job! Thank you.

File: src/core/indicators/pivot-point/PivotPointParameters.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        this._s2 = s2;
+        this._s1 = s3;
+    }
+}
+
+export class StandardPivotPoints extends PivotPoints {}
+
+export class FibonacciPivotPoints extends PivotPoints {}
+
+export class Candle {
```
[the remark this answers is not in the document]
Comment (Ahsan9981) [reply src/core/indicators/pivot-point/PivotPointParameters.ts:57]: ok got it. Let me fix that for you!

**Comment by Ahsan9981** [comment]
I  have pushed the requested change.

**Pull request closed without merging**